### PR TITLE
Changes emergency wall lockers to wall dispensers

### PR DESCRIPTION
**Pull request by comma**
Just a repath and new icon. They were already blocking all of closet functionality, so moved them up in the path where they belong.

## Changelog
:cl:
tweak: Wall emergency lockers renamed to emergency dispensers (since they're not lockers at all), and given new icon
/:cl:

### CODE REVIEW

File: code/game/objects/structures/emergency_dispenser.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	icon_state = "world"
+	anchored = TRUE
+	var/list/spawnitems = list(/obj/item/tank/emergency/oxygen,/obj/item/clothing/mask/breath)
+	var/amount = 3 // spawns each items X times.
+
+/obj/structure/emergency_dispenser/attack_hand(mob/user)
+	if(!CanPhysicallyInteract(user))	//Added by Strumpetplaya - AI shouldn't be able to         (you're both stupid, need CanPhysicallyInteract --Chinsky)
+		return									//activate emergency lockers.  This fixes that.  (Does this make sense, the AI can't call attack_hand, can it? --Mloc) 
+	if(!amount)
+		to_chat(usr, SPAN_WARNING("It's empty."))
```
Comment (afterthought2) [inline code/game/objects/structures/emergency_dispenser.dm]: `user`

File: code/game/objects/structures/emergency_dispenser.dm [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		return
+	if(amount)
+		to_chat(usr, SPAN_NOTICE("You take out some items from \the [src]."))
```
Comment (afterthought2) [inline code/game/objects/structures/emergency_dispenser.dm]: `user`

File: code/game/objects/structures/emergency_dispenser.dm [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		playsound(src, 'sound/machines/vending_machine.ogg', 25, 1)
+		for(var/path in spawnitems)
+			new path(src.loc)
+		amount--
+	return
```
Comment (afterthought2) [inline code/game/objects/structures/emergency_dispenser.dm]: terminal return

**Review by afterthought2 — COMMENTED** [review]
Are these constructable/deconstructable at all?

**Comment by comma** [comment]
Nope, not buildable at all. I guess they're deconstructible since they have default structure damage handling so can probably bash them down with a toolbox.

File: code/game/objects/structures/emergency_dispenser.dm (hunk 2 above)
Comment (comma) [reply code/game/objects/structures/emergency_dispenser.dm]: don

File: code/game/objects/structures/emergency_dispenser.dm (hunk 1 above)
Comment (comma) [reply code/game/objects/structures/emergency_dispenser.dm]: don

File: code/game/objects/structures/emergency_dispenser.dm (hunk 3 above)
Comment (comma) [reply code/game/objects/structures/emergency_dispenser.dm]: dokodon

**Comment by comma** [comment]
Also made spawnstuff list static

File: code/game/objects/structures/emergency_dispenser.dm [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	anchored = TRUE
+	var/static/list/spawnitems = list(/obj/item/tank/emergency/oxygen,/obj/item/clothing/mask/breath)
+	var/amount = 3 // spawns each items X times.
+
+/obj/structure/emergency_dispenser/attack_hand(mob/user)
+	if(!CanPhysicallyInteract(user))	//Added by Strumpetplaya - AI shouldn't be able to         (you're both stupid, need CanPhysicallyInteract --Chinsky)
+		return									//activate emergency lockers.  This fixes that.  (Does this make sense, the AI can't call attack_hand, can it? --Mloc) 
+	if(!amount)
+		to_chat(user, SPAN_WARNING("It's empty."))
+		return
```
Comment (MistakeNot4892) [inline code/game/objects/structures/emergency_dispenser.dm]: `return TRUE` to indicate the action completed.

File: code/game/objects/structures/emergency_dispenser.dm [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	desc = "A wall mounted dispenser with emergency supplies."
+	icon = 'icons/obj/structures/emergency_dispenser.dmi'
+	icon_state = "world"
+	anchored = TRUE
+	var/static/list/spawnitems = list(/obj/item/tank/emergency/oxygen,/obj/item/clothing/mask/breath)
+	var/amount = 3 // spawns each items X times.
+
+/obj/structure/emergency_dispenser/attack_hand(mob/user)
+	if(!CanPhysicallyInteract(user))	//Added by Strumpetplaya - AI shouldn't be able to         (you're both stupid, need CanPhysicallyInteract --Chinsky)
+		return									//activate emergency lockers.  This fixes that.  (Does this make sense, the AI can't call attack_hand, can it? --Mloc)
```
Comment (MistakeNot4892) [inline code/game/objects/structures/emergency_dispenser.dm]: pls either remove these comments or normalize the whitespace from tabs :(

File: code/game/objects/structures/emergency_dispenser.dm (hunk 4 above)
Comment (comma) [reply code/game/objects/structures/emergency_dispenser.dm]: donezo

File: code/game/objects/structures/emergency_dispenser.dm (hunk 5 above)
Comment (comma) [reply code/game/objects/structures/emergency_dispenser.dm]: aligned them

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**